cmd/check-status/main.go

```diff
@@ -15,10 +15,11 @@
 package main
 
 import (
-	"flag"
 	"fmt"
 	"os"
 
+	flag "github.com/spf13/pflag"
```

Comment:
What's the benefit of switching libraries here?

Reply:
The benefit here is that the use of `spf13/pflag` allows us to support both full-length and abbreviated command-line flags rather than forcing us to use just one. It also brands itself as a ["drop-in replacement for `flags`"](https://github.com/spf13/pflag#usage) so it should be safe? (famous last words...)